tools/amp_ner/ner_to_csv.xml

```diff
@@ -0,0 +1,22 @@
+<tool id="ner_to_csv" name="NER to CSV" version="1.1.0">
+  <description>Convert NER output to CSV</description>
```

Comment:
In a similar style  as the VTT generator, we can have the name as "AMP Named Entity JSON to CSV Converter"

Reply:
Done